src/main/java/net/mcreator/generator/template/TemplateGeneratorConfiguration.java

```diff
@@ -40,15 +40,19 @@ public TemplateGeneratorConfiguration(GeneratorConfiguration generatorConfigurat
 		templateLoaderPaths.addAll(generatorConfiguration.getImports());
 
 		List<TemplateLoader> templateLoaderList = new ArrayList<>();
-		for (String templateLoaderPath : templateLoaderPaths) {

```

Comment:
When you don't use extra mechanics of forEach, the for-each loop is actually more performant and cleaner so keep that if no additional reason to go with foreach